web/source/kmwbase.ts

```diff
@@ -147,7 +147,7 @@ namespace com.keyman {
       this['interface'] = this.core.keyboardInterface;
 
       this.modelManager = new text.prediction.ModelManager();
-      this.osk = this['osk'] = new com.keyman.osk.OSKManager();
+      this.osk = this['osk'] = null;
```

Comment:
Why both `this.osk` and `this['osk']`?

Reply:
`this['osk']` ensures that Google Closure doesn't minify the `osk` field name away entirely.  I wasn't sure (and actually, still wonder) how aggressively it minifies the code internally - whether or not Closure would pick a shorter, minified name for internal use for any `.osk` (instead of `['osk']`) references in the codebase - so just in case it does something that aggressive, the non-quoted version was placed there, too.

There's a fair shot it's unnecessary, and I probably should take the time to test that at some point.